tables/fr-bfu-comp6.utb

```diff
@@ -1,7 +1,7 @@
 # liblouis: Unified French 6 dots Braille table
 
 # Copyright (C) 1995-2004 by The BRLTTY Team
-# Copyright (C) 2012,1017 Michel Such <michel.such@free.fr>
+# Copyright (C) 2012 Michel Such <michel.such@free.fr>
```

Comment:
I wouldn't have changed this. Especially since he is behind the current changes. I can fix this when merging.

Reply:
Thanks, solved :)